couscous.yml

```diff
@@ -90,9 +90,9 @@ menu:
                 text: Custom domains
                 url: /docs/environment/custom-domains.html
                 title: Configuring custom domain names for HTTP applications
-            performances:
-                text: Performances
-                url: /docs/environment/performances.html
+            performance:
+                text: Performance
+                url: /docs/environment/performance.html
```

Comment:
Renaming the file would break existing links (outside of bref.sh itself), I think it might be better to avoid changing the URL/file name.